### PR TITLE
ES2015-ify

### PR DESCRIPTION
**Pull request by dciccale**
Arrow function + Template literals

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
-module.exports = function (url) {
+module.exports = (url) => {
 	if (typeof url !== 'string') {
-		throw new TypeError('Expected a string, got ' + typeof url);
+		throw new TypeError(`Expected a string, got ${typeof url}`);
```
Comment (SamVerschueren) [inline index.js]: I'd rather have the following

```js
throw new TypeError(`Expected \`url\` to be of type \`string\`, got \`${typeof url}\``);
```

@sindresorhus wdyt? I think this matches with how we do it lately with other modules.

**Comment by SamVerschueren** [comment]
Also change this line https://github.com/sindresorhus/prepend-http/blob/master/package.json#L13 and the `.travis.yml` file.

**Comment by sindresorhus** [comment]
And `.travis.yml`

**Comment by SamVerschueren** [comment]
Could you change the error message as well?

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**